pkg/http/server.go

```diff
@@ -112,10 +114,19 @@ func (s *server) useProxy(next fasthttp.RequestHandler) fasthttp.RequestHandler
 				ctx.RemoteAddr(),
 				ctx.Host(),
 				proto))
-		// Add X-Forwarded-For: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For
-		ctx.Request.Header.Add("X-Forwarded-For", ctx.RemoteAddr().String())
-		// Add X-Forwarded-Host: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host
-		ctx.Request.Header.Add("X-Forwarded-Host", fmt.Sprintf("%s", ctx.Host()))
+
```

Comment:
`Forwarded` needs to be changed. 

`Forwarded` is the standard proxy header. `X-*` is non-standard but widely used.

Reply:
Done